trpcultivate_phenotypes/src/Form/PhenodataBackupForm.php

```diff
@@ -0,0 +1,320 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Drupal\trpcultivate_phenotypes\Form;
+
+use Drupal\Core\Config\ConfigFactoryInterface;
+use Drupal\Core\Entity\EntityForm;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Session\AccountInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Drupal\tripal_chado\Controller\ChadoProjectAutocompleteController;
+
+/**
+ * Phenotypic Data Backup form.
+ */
+final class PhenodataBackupForm extends EntityForm {
+
+  /**
+   * File types supported.
+   */
+  private const array BACKUP_FILE_TYPES = [
+    'tsv',
```

Comment:
The share importer can only handle tsv so this makes sense... but I wonder if we should allow txt and xlsx formats here in recognition that while the data is being generated it may be in these other formats?

Reply:
After a discussion with Carolyn we decided to move forward with expanding the formats supported. This was done in 2f389f84646eb084f9956dc6bd9b545a868f0b92.